app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+                  class="project-btn project-btn-3 active-btn-effect hover-btn-effect"
+                  >${recentProjects[5].workStack[2]}</a
+                >
+              </li>
+            </ul>
+          `;
+    }
+
+    if (desktopProjectDiv.innerHTML !== '') {
+      descktopProjectsWrapperDiv.appendChild(desktopProjectDiv);
+    }
+  }
+}
+
+// portfolio end
+
+// popup start
+
+const modal = document.querySelector('.project-details-popup');
+const overlay = document.querySelector('.overlay');
+const openModalBtn = document.querySelectorAll('.pop');
+
+overlay.classList.add('hidden');
+const popupContainer = document.getElementById('project-details-popup');
+popupContainer.innerHTML = ` <div class="project-header">
+        <h3 class="project-title-popup">Multi-Post Stories</h3>
+        <i class="fa fa-close popup-close" id="close-x"></i>
+      </div>
+      <div>
+        <ul class="project-technology-popup">
+          <li>
+            <a
+              href="https://html.com/"
+              class="project-btn-pop project-btn-1-pop active-btn-effect hover-btn-effect"
+            >
+              HTML</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://getbootstrap.com/"
+              class="project-btn-pop project-btn-2 active-btn-effect hover-btn-effect"
+              >Bootstrap</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://www.ruby-lang.org/en/"
+              class="project-btn-pop project-btn-3 active-btn-effect hover-btn-effect"
+              >Ruby on Rails</a
+            >
+          </li>
+        </ul>
+      </div>
+      <div class="project-more">
+        <div>
+          <img
+            src="images/Snapshoot Portfolio.png"
+            alt=""
+            class="popup-project-img"
+          />
+        </div>
+
+        <div class="popup-project-more-details">
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not
+            only five centuries, but also the leap into electronic typesetting,
+            remaining essent
+          </p>
+          <ul class="popup-project-buttons">
+            <li class="first-child">
+              <a href=""
+                >See live<img
+                  src="images/live-Icon.png"
+                  alt="Live-icon"
+                  class="popup-project-button-icon-1"
+              /></a>
+            </li>
+            <li class="last-child">
+              <a href=""
+                >See source<img
+                  src="images/Icon -GitHub-white.png"
+                  alt="Live-icon"
+                  class="popup-project-button-icon-2"
+              /></a>
+            </li>
```

Comment:
- [ ] As mentioned earlier, the links in the project cards should be dynamic. You can achieve this by using the project array attributes and adding a dynamic href attribute to each link. This way, each project will have its own live and source link, providing a more personalized experience for users.
- [ ] Furthermore, I noticed that the text and icon alignment in the project cards doesn't match the design. To ensure consistency with the design, please update your code accordingly and align the text and icons correctly.
- [ ] Additionally, to enhance user interaction, consider adding hover and pressed states for your buttons as specified in the design. These states will provide visual feedback to users and improve the overall user experience.
![image](https://github.com/FilbertKimasa/Portfolio/assets/55840999/5686462d-70c9-49cf-9051-8dd527a800bb)

By making these updates, your project will better align with the design requirements and offer a more polished and professional look. Keep up the great work! 🚀💻🌟